ONSdigital/census-rm-qid-batch-runner
# Update unaddressed batch config CSV

# Motivation and Context
The unaddressed batch requirements have changed, the config here needs updating to match

# What has changed
* Update CSV unaddressed batch config

# How to test?
* Double check the config changes against the requirements: https://collaborate2.ons.gov.uk/confluence/display/SDC/Unaddressed+Print+Files
* Run through with the new config and check the unaddressed batch files/queue backup dump still works as expected.

# Links
https://trello.com/c/CUNdywUZ/872-update-unaddressed-batch-config-3
https://collaborate2.ons.gov.uk/confluence/display/SDC/Unaddressed+Print+Files
